README.md

```diff
@@ -68,17 +68,17 @@ A selection wider than [recommended fonts](#recommended-fonts) are included by d
 
 ## All Available Fonts
 
-Any of the below fonts may be included with `o-fonts` using [SASS](#sass). Build Service users are limited to [fonts included by default](#fonts-included-by-default).
-
-| Weight   | FinancierDisplayWeb | MetricWeb |
-|----------|:-------------------:|:---------:|
-| thin     |                     |    ✓      |
-| light    |           i         |    ✓ i    |
-| regular  |         ✓ i         |    ✓ i    |
-| medium   |         ✓ i         |    ✓      |
-| semibold |           i         |    ✓      |
-| bold     |         ✓           |    ✓ i    |
-| black    |                     |           |
+Any of the below fonts may be included with `o-fonts` using [SASS](#sass), this is usual for brands such as specialist titles who may use additional fonts. Build Service users are limited to [fonts included by default](#fonts-included-by-default).
```

Comment:
@notlee, I can't tell if this is meant to say "usual" or "usually". I think both read well but wanted to check if you meant to write "usually" instead?
>Any of the below fonts may be included with `o-fonts` using [SASS](#sass), this is usual for brands such as specialist titles who may use additional fonts.

Reply:
It's supposed to say "useful'? 🙈

Reply:
```suggestion
Any of the below fonts may be included with `o-fonts` using [SASS](#sass), this is useful for brands such as specialist titles who may use additional fonts. Build Service users are limited to [fonts included by default](#fonts-included-by-default).
```